zygrader/ui/themes.py

```diff
@@ -0,0 +1,114 @@
+"""Defines all the color pairs and emojis for each theme.
+To add a new theme, use 'curses.init_pair()' with the 256-colors you would like (use constants).
+
+"""
+
+import curses
+THEMES = {
+    "Default": True,
+    "Christmas": False,
+    "Spooky": False,
+    "Birthday": False
+}
+
+
+class Theme:
+    def __init__(self):
+        self.__init_colors()
+        # cannot decalre this outside the function, since it requires curses to be initialized
+        self.THEME_COLORS = {
```

Comment:
That makes sense. In this case, I would prefer `lower_snake_case` since these aren't static class data.